Gemfile

```diff
@@ -4,6 +4,7 @@ plugin 'bundler-inject', '~> 1.1'
 require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundler-inject") rescue nil
 
 gem "activesupport", '~> 5.2.4.3'
+gem "clowder-common-ruby" # version specified in providers-common
```

Comment:
So even if providers-common pulls in clowder-common-ruby we still need to declare it separately in the Gemfile?

Reply:
it's not needed, but we talked about it some time ago that we'll declare it on both sides